common/test_rescue.yml

```diff
@@ -76,21 +76,27 @@
         - "vmware.log"
       when: vm_dir_name is defined and vm_dir_name
 
-    - name: "Check Windows guest BSOD in vmware.log"
-      when:
-        - gosv_test_suite == 'windows'
-        - datastore_file_download_success
+    - name: "Handle Windows guest OS situation"
+      when: gosv_test_suite == 'windows'
       block:
-        - name: "Look up Windows guest BSOD in vmware.log"
+        - name: "Set exit testing when meets VMCI known issue"
           ansible.builtin.set_fact:
-            winbsod_in_vmware_log: "{{ lookup('file', datastore_file_download_result.dest) | regex_findall('.*WinBSOD:.*') }}"
+            exit_testing_when_fail: true
+          when: vmci_known_issue | default(false)
```

Comment:
It's better to put this task into the test case not this common task. Otherwise, all failed test cases will check the value of vmci_known_issue.

Reply:
Since this failure would happen in the test setup in any test case, and if this known issue happens, there is no need to run the following test cases.

Reply:
I mean set ```exit_testing_when_fail: true``` in ```handle_vmci_known_issue.yml```  when vmci_known_issue is set to true . After all, the known issue does not happen on all ESXi  servers.